x-pack/plugins/ml/public/application/model_management/deployment_setup.tsx

```diff
@@ -67,300 +128,357 @@ export const DeploymentSetup: FC<DeploymentSetupProps> = ({
+            defaultMessage="Advanced Configurations"
+          />
+        }
+        initialIsOpen={isUpdate}
+      >
+        <EuiSpacer size={'m'} />
+
+        <EuiPanel hasBorder hasShadow={false}>
           <EuiFormRow
+            hasChildLabel={false}
+            fullWidth
             label={
               <FormattedMessage
-                id="xpack.ml.trainedModels.modelsList.startDeployment.threadsPerAllocationLabel"
-                defaultMessage="Threads per allocation"
+                id="xpack.ml.trainedModels.modelsList.startDeployment.vCpuUsageLabel"
+                defaultMessage="vCPUs usage level"
               />
             }
-            hasChildLabel={false}
-            isDisabled={disableThreadingControls}
           >
-            <EuiButtonGroup
-              isDisabled={disableThreadingControls}
-              legend={i18n.translate(
-                'xpack.ml.trainedModels.modelsList.startDeployment.threadsPerAllocationLegend',
-                {
-                  defaultMessage: 'Threads per allocation selector',
-                }
+            <EuiRange
+              fullWidth
+              id={'vCpuLevel'}
+              min={0.5}
+              max={2.5}
+              step={1}
+              tickInterval={1}
+              value={vCpuLevelMap[config.vCPUUsage].value}
+              onChange={(e) => {
+                const result = Object.entries(vCpuLevelMap).find(
+                  ([, val]) => val.value === Number(e.currentTarget.value)
+                );
+                onConfigChange({
+                  ...config,
+                  vCPUUsage: result![0]! as DeploymentParamsUI['vCPUUsage'],
+                });
+              }}
+              showTicks
+              ticks={customTicks}
+              levels={customColorsLevels}
+              aria-label={i18n.translate(
+                'xpack.ml.trainedModels.modelsList.startDeployment.vCpuLevel',
+                { defaultMessage: 'vCPUs level selector' }
               )}
-              name={'threadsPerAllocation'}
-              isFullWidth
-              idSelected={
-                disableThreadingControls
-                  ? '1'
-                  : threadsPerAllocationsOptions.find((v) => v.value === threadsPerAllocations)!.id
+              aria-describedby={'vCpuRangeHelp'}
+            />
+          </EuiFormRow>
+
+          <EuiSpacer size={'s'} />
+
+          <EuiFormHelpText id={'vCpuRangeHelp'}>
+            <EuiCallOut size="s">
+              <p>{helperText}</p>
+            </EuiCallOut>
+          </EuiFormHelpText>
+        </EuiPanel>
+
+        {!disableAdaptiveResourcesControl ? (
+          <>
+            <EuiSpacer size={'m'} />
+
+            <EuiSwitch
+              label={
+                <FormattedMessage
+                  id="xpack.ml.trainedModels.modelsList.startDeployment.adaptiveAllocationEnabledLabel"
+                  defaultMessage="Adaptive Resources (recommended)"
```

Comment:
Should this be sentence case here `Adaptive resources` ?

Reply:
Fixed in c5f7e84cd69e880bca8c5aad5d345ff14d31c4fd